helm/examples/azure/sa/create-secret.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+
+[[ -n "$SUBSCRIPTION" ]] && az account set --subscription $SUBSCRIPTION
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SECRET_NAME_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SHARE_NAME_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*shareName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SECRET_NAMESPACE_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretNamespace:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+[[ -n "$SECRET_NAME_INPUT" ]] && SECRET_NAME=${SECRET_NAME_INPUT}
+[[ -n "$SECRET_NAMESPACE_INPUT" ]] && SECRET_NAMESPACE=${SECRET_NAMESPACE_INPUT}
+[[ -n "$SHARE_NAME_INPUT" ]] && SHARE_NAME=${SHARE_NAME_INPUT}
+
+if [[ -z "$SECRET_NAME" ]] || [[ -z "$SECRET_NAMESPACE" ]] || [[ -z "$SHARE_NAME" ]]
+then
+  echo "Miss one of SECRET_NAME, SECRET_NAMESPACE and SHARE_NAME"                                                                           exit 1
```

Comment:
Formatting:  'exit 1' is on the same line as the echo, spaced far to the right. It should be on the following line.

Reply:
fixed